queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01'; 
```

Comment:
Great job @IsmaelMastronardi 🤝

- [ ] As stated in the project requirements, kindly ensure your queries are accurate as provided the date_of birth should be between ``2016/01/01`` and ``2019/31/12`` instead of `2016-01-01` and `2019-01-01`.
- [ ] Kindly remember to update your screenshots for the updated queries.